_sass/_base.scss

```diff
@@ -6,3 +6,7 @@ body {
 img {
   max-width: 100%;
 }
+
+.this__is-a--test {
+  color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.